src/components/AddItem/AddItem.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { Button, Modal, Form } from 'semantic-ui-react'
+
+const AddItem = () => {
+  const [open, setOpen] = React.useState(false)
+
+  return (
+    <Modal
+      onClose={() => setOpen(false)}
```

Comment:
perhaps we can extract this into a function inside the component that describes what is happening in the function name.